python/GafferImageUI/RGBAChannelsPlugValueWidget.py

```diff
@@ -100,14 +102,28 @@ def __menuDefinition( self ) :
 				text = prefix + "RGBA"
 				value = [ prefix + x for x in [ "R", "G", "B", "A" ] ]
 			else :
+				nonStandardLayers.add( text.split( "." )[0] )	
 				text = channelName
 				value = [ channelName ] * 4
 
 			if text in added :
 				continue
 
+			added.add( text )
+			channelOptions.append( ( text, value ) )
+
+		for text, value in channelOptions:
+			menuName = "/" + text
+
+			if text.split(".")[0] in nonStandardLayers: 
+				# If there are non-standard channels in this layer, then make a submenu for it
+				menuName = menuName.replace( ".", "/" )
+			else:
+				# If there are only the standard channels, we don't need a submenu
+				menuName = menuName.replace( ".RGBA", "" )
```

Comment:
On this branch we're not replacing "." with "/", so when we have stereo images (or any other nested layers), we no longer making a submenu for the top-level left/right split. Is this intentional? The menu is already getting pretty long with a single view, so I think that keeping each view on a submenu would probably be for the best.